### PR TITLE
Windup 2332 - Application List Filtering/Labelling to support Target Runtime Selection

### PR DESCRIPTION
**Pull request by carlosthe19916**
https://issues.jboss.org/browse/WINDUP-2332
This PR adds the default label file configuration to detect Target Runtimes

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

File: rules-reviewed/core.windup.label.xml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                <tag>JCA</tag>
+                <tag>JMS</tag>
+                <tag>JMS Connection Factory</tag>
+                <tag>JMS Queue</tag>
+                <tag>JMS Topic</tag>
+                <tag>JPA</tag>
+                <tag>JPA entities</tag>
+                <tag>JPA named queries</tag>
+                <tag>JPA XML</tag>
+                <tag>JSON-B</tag>
```
Comment (m-brophy) [inline rules-reviewed/core.windup.label.xml]: Is JSON-B unsupported?

File: rules-reviewed/core.windup.label.xml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                <tag>Message (MDB)</tag>
+                <tag>RMI</tag>
+                <tag>SOAP (SAAJ)</tag>
+                <tag>Stateful (SFSB)</tag>
+                <tag>Stateless (SLSB)</tag>
+                <tag>WS Metadata</tag>
+            </unsuitable>
+            <neutral>
+                <tag>Java Source</tag>
+                <tag>JDBC datasources</tag>
```
Comment (m-brophy) [inline rules-reviewed/core.windup.label.xml]: Is JDBC datasources unsupported?

**Comment by mrizzi** [comment]
retest

File: rules-reviewed/core.windup.label.xml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                <tag>JACC</tag>
+                <tag>Java EE Batch</tag>
+                <tag>Java EE Batch API</tag>
+                <tag>Java EE JSON-P</tag>
+                <tag>Java EE Security</tag>
+                <tag>JavaMail</tag>
+                <tag>JAX-RPC</tag>
+                <tag>JAX-RS</tag>
+                <tag>JAX-WS</tag>
+                <tag>JAXB</tag>
```
[the remark this answers is not in the document]
Comment (carlosthe19916) [reply rules-reviewed/core.windup.label.xml]: You're right, not it is supported

File: rules-reviewed/core.windup.label.xml (hunk 1 above)
Comment (carlosthe19916) [reply rules-reviewed/core.windup.label.xml]: Ass agreed in our last meeting JSON-B will be supported by EAP and on JWS it won't be either supported, unsupported, or neutral

File: rules-reviewed/core.windup.label.xml (hunk 2 above)
Comment (carlosthe19916) [reply rules-reviewed/core.windup.label.xml]: `JDBC datasources` has been moved to neutral in EAP and JWS

File: rules-reviewed/core.windup.label.xml [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                <tag>Clustering Web Session</tag>
+                <tag>Common Annotations</tag>
+                <tag>EAR</tag>
+                <tag>EJB</tag>
+                <tag>EJB XML*</tag>
+<!--                <tag>EJB XML</tag>-->
+<!--                <tag>EJB XML 3.1</tag>        &lt;!&ndash; temp &ndash;&gt;-->
+                <tag>Enterprise Web Services</tag>
+                <tag>JACC</tag>
+                <tag>Java EE Batch</tag>
```
[the remark this answers is not in the document]
Comment (carlosthe19916) [reply rules-reviewed/core.windup.label.xml]: tag added, thanks

**Comment by mrizzi** [comment]
add to whitelist

File: rules-reviewed/core.windup.label.xml [hunk 5]
```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<labelset id="core_labels"
+          priority="1"
+          xmlns="http://windup.jboss.org/schema/jboss-labelset"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://windup.jboss.org/schema/jboss-labelset http://windup.jboss.org/schema/jboss-labelset/windup-jboss-labelset.xsd">
+    <labels>
+        <label id="eap">
+            <name>EAP</name>
```
Comment (mrizzi) [inline rules-reviewed/core.windup.label.xml]:
```suggestion
            <name>JBoss EAP</name>
```

File: rules-reviewed/core.windup.label.xml [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+            <description>Red Hat Jboss Enterprise Application Platform</description>
```
Comment (mrizzi) [inline rules-reviewed/core.windup.label.xml]:
```suggestion
            <description>Red Hat JBoss Enterprise Application Platform</description>
```

File: rules-reviewed/core.windup.label.xml [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+                <tag>Spring MVC (embedded)</tag>      <!-- Spring MVC is supported -->
+                <tag>Spring Security (embedded)</tag> <!-- Spring Security is supported -->
+                <tag>Swagger (embedded)</tag>
+                <tag>Web XML*</tag>
+                <tag>WSDL (embedded)</tag>
+            </neutral>
+        </label>
+        <label id="jws">
+            <name>JWS</name>
+            <description>Red Hat Jboss Web Server</description>
```
Comment (mrizzi) [inline rules-reviewed/core.windup.label.xml]:
```suggestion
            <description>Red Hat JBoss Web Server</description>
```

**Pull request merged**